### PR TITLE
README/Composer: suggest the Requests PSR18 adapter package

### LINKED ISSUE #320: Question: Are there near-term plans for PSR-7 Message Interface compatibility?
Are there near-term plans for PSR-7 Message Interface compatibility?

I'm developing some new software which leverages PSR-7 pretty heavily, and I'm wanting to write some tutorials for how to use it alongside Guzzle, Buzz, and Requests.

I know that Requests pre-dates PSR-7 by many years, but I didn't know if you had plans for a PSR-7 update.

### PR DESCRIPTION
**Pull request by jrfnl**
## Pull Request Type

- [x] I have checked there is no other PR open for the same change.

This is a:
- [ ] Bug fix
- [ ] New feature
- [x] Documentation improvement
- [ ] Code quality improvement

## Context

Recommend @art4's package for a PSR-7/PSR-18 compatible version of Requests.

Closes #768
Closes #320

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,6 +141,24 @@ set of tests. We're not quite there yet, but [we're getting close][codecov].
...
 
 
+Requests and PSR-7/PSR-18
+-------------------------
+
+[PSR-7][psr-7] describes common interfaces for representing HTTP messages.
+[PSR-18][psr-18] describes a common interface for sending HTTP requests and receiving HTTP responses.
+
+Both PSR-7 as well as PSR-18 were created after Requests' conception.
+At this time, there is no intention to add a native PST-7/PSR-18 implementation to the Requests library.
```
Comment (Art4) [inline README.md]:
```suggestion
At this time, there is no intention to add a native PSR-7/PSR-18 implementation to the Requests library.
```

File: README.md [hunk 2]
```diff
@@ -141,6 +141,24 @@ set of tests. We're not quite there yet, but [we're getting close][codecov].
...
+
+However, the amazing [Artur Weigandt][art4] has created a [package][requests-psr-18], which allows you to use Requests as a PSR-7 compatible PSR-18 HTTP Client.
```
Comment (Art4) [inline README.md:153]:
```suggestion
However, [Artur Weigandt][art4] has created a [package][requests-psr-18], which allows you to use Requests as a PSR-7 compatible PSR-18 HTTP Client.
```

**Review by Art4 — CHANGES_REQUESTED** [review]
Thank you. :blush:

File: README.md (hunk 2 above)
Comment (jrfnl) [reply README.md:153]: @Art4 Are you sure ? I wrote it cause I mean it. You've done an amazing job creating that package, so I'm happy for us to warmly endorse it.

Comment (Art4) [reply README.md:153]: @jrfnl Sorry, I'm very bad at getting compliments. Please continue as you like.

**Review by Art4 — APPROVED** [review]

**Review by schlessera — APPROVED** [review]

**Pull request merged**